Cargo.toml

```diff
@@ -6,7 +6,7 @@ license = "MIT/Apache-2.0"
 readme = "README.md"
 repository = "https://github.com/rust-lang-nursery/rand"
 documentation = "https://docs.rs/rand"
-homepage = "https://github.com/rust-lang-nursery/rand"
+homepage = "https://crates.io/crates/rand"
```

Comment:
I don't like either :smile:. Every time I look up a create on crates.io is surprisingly unhelpful, and I quickly click through to the repository or docs.rs.

But this seems like a good change.